iocore/net/UnixNetVConnection.cc

```diff
@@ -1247,6 +1247,7 @@ UnixNetVConnection::populate(Connection &con_in, Continuation *c, void *arg)
   }
 
   if (h->startIO(this) < 0) {
+    con_in.move(this->con);
     Debug("iocore_net", "populate : Failed to add to epoll list");
```

Comment:
I assume this con_in.move was added to ensure the socket get's cleaned up?

Reply:
If startIO failed, move the con.fd back to con_in.fd. Avoid to close the con_in.fd.